src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaFigure, DrawFigure {
+    private float radius = 1;
```

Comment:
private fields mustn't be initialized here and you don't need to initialize it with default number, the same with other figures)